### PR TITLE
Feature/packages/people #cmx 129

### PR DESCRIPTION
**Pull request by rebecatalkdesk**
- Adding new People Section component

### CODE REVIEW

File: packages/organisms/people/src/People.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import PeopleBig from './PeopleBig';
+
+const People = ({ title, subtitle, people_component: people, ...section }: Props) => {
+  section = {
+    ...section,
+    childSectionClass: Defaults.childSectionClass,
+  };
+  return (
+    <Section {...section}>
+      {people.map((person: PeopleProps, idx: number) => {
```
Comment (Mar0cas) [inline packages/organisms/people/src/People.tsx:15]: For precaution change to `people?.map(...)`.

File: packages/organisms/people/src/People.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          return <PeopleBig key={`person-b-${idx}`} {...person} title={title} />;
+      })}
+      <div className="row justify-content-around">
+        {!title || (
+          <div className={'col-14 section__people--content text-center mb-5'}>
+            <Heading level={2}>{title}</Heading>
+            {!subtitle || <Heading level={3}>{subtitle}</Heading>}
+          </div>
+        )}
+        {people.map((person: PeopleProps, idx: number) => {
```
Comment (Mar0cas) [inline packages/organisms/people/src/People.tsx:26]: Same as above.

File: packages/organisms/people/src/PeopleSmall.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  bio,
+  image,
+  social_networks_component: social,
+  title,
+}: PeopleProps) => {
+  const contentObject: ContentProps = { text: bio, class: '-small' };
+  return (
+    <div className="col-14 col-sm-6 col-md-5 col-lg-4 col-xl-3 section__people--small">
+      <div className="item__media col-8 col-sm-6 col-md-8 col-lg-7 col-xl-7">
+        {!image || <Image {...image} alt={title} />}
```
Comment (Mar0cas) [inline packages/organisms/people/src/PeopleSmall.tsx:20]: Normally the [image scheme](https://github.com/Talkdesk/CREAC/blob/master/packages/organisms/panel/src/__stories__/__mock__/splitInfoRight.json#L19) has his own alt title, this is applied by the SEO team, don't use the page title.

File: packages/organisms/people/src/PeopleBig.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  social_networks_component: social,
+  side_class,
+  title,
+}: PeopleProps) => {
+  const contentObject: ContentProps = { text: bio, class: '-small' };
+  return (
+    <div className="section__people--big">
+      <div className="row align-items-center">
+        <div className={`col-14 col-sm-7 col-md-6 col-lg-6 col-xl-6 ${side_class?.media}`}>
+          <div className="people__media">{!image || <Image {...image} alt={title} />}</div>
```
Comment (Mar0cas) [inline packages/organisms/people/src/PeopleBig.tsx:22]: Normally the [image scheme](https://github.com/Talkdesk/CREAC/blob/master/packages/organisms/panel/src/__stories__/__mock__/splitInfoRight.json#L19) has his own alt title, this is applied by the SEO team, don't use the page title.

File: packages/organisms/people/src/component/Utils.ts [hunk 5]
```diff
@@ -0,0 +1 @@
+/* People utils here... */
```
Comment (Mar0cas) [inline packages/organisms/people/src/component/Utils.ts:1]: Not in use, delete.

File: packages/organisms/people/src/PeopleSmall.tsx (hunk 3 above)
Comment (rebecatalkdesk) [reply packages/organisms/people/src/PeopleSmall.tsx:20]: This is how it is in the twig version, with the title as the alt.

File: packages/organisms/people/src/PeopleBig.tsx (hunk 4 above)
Comment (rebecatalkdesk) [reply packages/organisms/people/src/PeopleBig.tsx:22]: This is how it is in the twig version, with the title as the alt.

**Pull request merged**